keyboards/candykeys/candy87/readme.md

```diff
@@ -0,0 +1,29 @@
+# Candy87
+
+![Candy87](https://i.imgur.com/HeCz1u8.png)
+
+The Candy87 Hotswap PCB is a universal TKL PCB by Candykeys.
+It supports ISO Hotswap by default (ANSI possible with desolering two hotswap sockets) and many layout options. 
```

Comment:
> and many layout options. 

So why is only a single layout defined within the json config?

Reply:
This is one "ALL" layout, so it has all of the different layouts and keys in this one layout.
On some keyboards these layouts were named "LAYOUT_ALL". It just has so many combination of layouts that this "ALL" layout is more convenient

Reply:
I think you are misunderstanding the question, and trying to explain something that I, a maintainer of QMK, are already aware of.

Having a single "all" layout macro is invalid. It not being called `LAYOUT_all` to avoid the lint warning is also invalid

You need to add more layouts.

Reply:
Please apologize. I added many (community) layouts to it now

Reply:
Added keymaps for all layouts - should be fine now